resource_obmcs_identity_policy.go

```diff
@@ -69,6 +85,27 @@ func PolicyResource() *schema.Resource {
 	}
 }
 
+func ignorePolicyFormatDiff() schema.SchemaDiffSuppressFunc {
+	return func(k string, old string, new string, d *schema.ResourceData) bool {
+		oldHash := getOrDefault(d, "policyHash", "")
+		newHash := getMD5Hash(toStringArray(d.Get("statements")))
+		oldETag := getOrDefault(d, "lastUpdateETag", "")
+		currentETag := getOrDefault(d, "ETag", "")
+		suppressDiff := strings.EqualFold(oldHash, newHash) && strings.EqualFold(oldETag, currentETag)
+
+		return suppressDiff
+	}
+}
```

Comment:
Why not just implement SchemaDiffSuppressFunc with the ignorePolicyFormatDiff function and pass it to the schema directly?

Reply:
done